easybuild/easyconfigs/d/datamash/Datamash-1.3-foss-2018a.eb

```diff
@@ -0,0 +1,24 @@
+# Author: Piotr Malicki
+
+easyblock = 'ConfigureMake'
+
+name = 'Datamash'
+version = '1.3'
+
+homepage = 'https://www.gnu.org/software/datamash/'
+description = "GNU datamash is performs basic numeric, textual and statistical operations on input data files"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://ftp.gnu.org/gnu/datamash/']
+sources = ['datamash-%(version)s.tar.gz']
+
+checksums = [
+    '47d382090e367ddb4967d640aba77b66'
```

Comment:
Please make this a SHA256 checksum, and make it a single line, there's no point in spreading this across multiple lines.

`eb --inject-checksums` will do this for you